src/test/java/it/pagopa/pn/delivery/PnSentDeliveryControllerTest.java

```diff
@@ -0,0 +1,99 @@
+package it.pagopa.pn.delivery;
```

Comment:
Un buon articolo che spiega come fare i test dei controller in modo che inizializzino lo stretto necessario.
https://reflectoring.io/spring-boot-web-controller-test/

Reply:
test implementato con Spring WebFlux e api WebTestClient
https://reflectoring.io/spring-webclient/